metrics/goplugin.go

```diff
@@ -0,0 +1,10 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var GopluginContractExecutionTime = prometheus.NewSummary(prometheus.SummaryOpts{
+	Name:       "goplugin_contract_execution_time",
+	Help:       "Time spent on execution contract, measured in goplugin",
+	Namespace:  insolarNamespace,
+	Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.05, 0.95: 0.05, 0.99: 0.05},
```

Comment:
values in this map are incorrect, use DefObjectives as starting point

Reply:
fixed